### DISCUSSION TITLE
Electricity generation forecast for the next 24 hours

### DISCUSSION THREAD

**Discussion by comatrixco** [electricitymaps/electricitymaps-contrib#2649] [General]
Hello,
 I am a regular user of your application and it helps me to know how I can consume cleaner energy.

 Would it be possible to add a section to know what the "theoretical" energy production will be for the next day?

 Electricity by type of production is auctioned the day before, in Spain on this page you can see, with these data, you can make an estimate of what the energy mix will be the next day.

 [OMIE SPAIN](https://www.omie.es/es/market-results/daily/daily-market/hourly-power-technologies?scope=daily&date=2020-09-11&system=1) 

 I suppose that other countries will have the same or similar system.

 Kind regards and thank you for helping a cleaner energy system;).

Hola,
Soy un usuario habitual de vuestra aplicación y me ayuda a conocer como puedo consumir energía más limpia.

¿Sería posible añadir apartado para conocer cual va a ser la producción de energía "teórica" del día siguiente?

La electricidad por tipo de producción es subastada el día anterior, en España en esta página se puede ver, con estos datos, se puede hacer una estimación de cual va a ser el mix energético del día siguiente. 

https://www.omie.es/es/market-results/daily/daily-market/hourly-power-technologies?scope=daily&date=2020-09-11&system=1

Supongo que otros países tendrán el mismo sistema o parecido.

Un cordial saludo y gracias por ayudar a un sistema energético más limpio ;).

Salud y Buena Energía
![IMG_20200911_173301_113](https://user-images.githubusercontent.com/71141704/92944946-eaaf2b00-f454-11ea-9bc8-9af225107760.jpg)
![IMG_20200911_173254_703](https://user-images.githubusercontent.com/71141704/92944955-ed118500-f454-11ea-9042-859c19aa2dcd.jpg)

**Comment by Kongkille** [comment]
Hi @comatrixco,

Thanks for bringing this up. On top of the electricity map, Tomorrow also has an API as a product which includes a forecast of the data, including the generation mix (based on machine learning). As such, we would have to consider whether we want to make that data public, there are numerous tradeoffs to this, as you can imagine, including potential changes to the UI to properly display to users what the current values are and which are forecasts.

I will label this issue as "Idea" and leave it open, although I do not think this is something we should act on right now.

Finally, can you perhaps rename this issue into English, just so everyone can understand what it's about?  

Thanks!

**Comment by comatrixco** [comment]
Thank you for your response,  i try to change the subjet un english ;).
